runtime/cudaq/platform/mqpu/remote-sim.config

```diff
@@ -0,0 +1,53 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+msg=""
+
+# Tell NVQ++ to generate glue code to set the target backend name
+GEN_TARGET_BACKEND=true
+
+# Use the MultiQPUPlatform
+PLATFORM_LIBRARY=mqpu
+
+COMPILER_FLAGS="$COMPILER_FLAGS -DCUDAQ_REMOTE_SIM"
+
+if ${LIBRARY_MODE}; then
+	COMPILER_FLAGS="$COMPILER_FLAGS -fembed-bitcode=all"
```

Comment:
Does this work if `CXX=gcc`? In this case, I'm specifically asking about `-fembed-bitcode`, but I guess the question really applies to the whole thing (i.e. the full REST Server-Client platform PR). My initial thought would've been that this is not a library mode thing since it requires code injection, etc., but line 19 says `if ${LIBRARY_MODE}`, so now I'm not sure.

Reply:
No, it won't work. It is supposed to work with the default `clang` compiler shipped with CUDA Quantum only.
I've added a validation for that condition in https://github.com/NVIDIA/cuda-quantum/pull/1012/commits/b370fa065429d434bbf03363e64315ba11536c41